suite/tests/runall.cmake

```diff
@@ -122,6 +124,30 @@ function (do_sleep ms)
   endif ()
 endfunction (do_sleep)
 
+function (wait_to_attach pid)
```

Comment:
Wouldn't this logic be better off in a separate .c file? cmake doesn't feel like the right place. Or is there some reason I'm missing?

Reply:
We used to have shell scripts and other scripts and had portability issues on Windows; we settled on cmake as our cross-platform scripting solution, though it can be a little awkward to use.